rootfs/templates/wrapper

```diff
@@ -89,6 +89,11 @@ function use() {
     if [ "${HOME}" == "/localhost" ]; then
       echo "WARNING: not mounting ${HOME} because it conflicts with geodesic"
     else
+      if [[ $(uname -r) =~ Microsoft$ ]]; then
```

Comment:
Let's use the fix here: https://github.com/cloudposse/geodesic/issues/199#issuecomment-409610973

Reply:
Let's swap out `sed` with `tr`